zebra/zebra_fpm_netlink.c

```diff
@@ -169,9 +169,12 @@ static int netlink_route_info_add_nh(struct netlink_route_info *ri,
 	struct interface *ifp = NULL, *link_if = NULL;
 	struct zebra_if *zif = NULL;
 	vni_t vni = 0;
+	extern struct in_addr ipv4_ll;
```

Comment:
we are planning on further separating out the different data planes.  Creating this cross dependency is not the way I would prefer we go forward here.  the fpm code should not be accessing data/values in another data plane module.  Especially if someone tried to use this fpm_netlink code on bsd.

I would recommend just creating another variable inside this module to copy the rt_netlink.c behavior.

Reply:
Sure. Will make the change and re-submit